### PR TITLE
FOGL-4241 default value and description fixes

### PR DESCRIPTION
**Pull request by praveen-garg**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -3,34 +3,51 @@ Fledge South Person detection
...
         model, this file should also be placed in same directory as the model.
 
 Asset Name:
         The name of the asset used for the readings generated by this plugin.
 
-Enable Edge TPU :
-        If you want to use Coral Edge TPU then enable this config parameter. Also
-        change the name of the model file to detect_edgetpu.tflite
+Enable Edge TPU:
+        If you don't want to use Coral Edge TPU then disbale this configuration parameter.
```
Comment (deepanshu-yadav) [inline README.rst:29]: *disable

File: README.rst [hunk 2]
```diff
@@ -3,34 +3,51 @@ Fledge South Person detection
 ******************************
 
 This directory contains a South service plugin that a detects person in the live video stream
```
Comment (AmandeepArora) [inline README.rst:5]: stray "a"

File: README.rst (hunk 1 above)
Comment (AmandeepArora) [reply README.rst:29]: Language should be "Indicates whether to use edge TPU for inference".

File: README.rst [hunk 3]
```diff
@@ -56,10 +73,10 @@ Instructions For Installation
                 Size: Discrete 2592x1944
                         Interval: Discrete 0.067s (15.000 fps)
                 Size: Discrete 0x0
+
 3. You can choose resolution from here.
 
-4. Camera ID is mostly 0, but you can replace video0 with video1
-in above command.
+4. Camera ID is mostly 0, but you can replace video*0* with video*1* in above command.
```
Comment (AmandeepArora) [inline README.rst:79]: It should be "Above example uses Camera ID 0 to indicate use of /dev/video0 device, please use the applicable value for your setup."

File: python/fledge/plugins/south/person_detection/person_detection.py [hunk 4]
```diff
@@ -59,28 +59,28 @@ def check_background():
...
+        'description': 'Asset name',
         'type': 'string',
-        'default': 'Detection Results',
+        'default': 'person_detection',
         'order': '3',
         'displayName': 'Asset Name'
     },
     'enable_edge_tpu': {
-        'description': 'Connect the Coral Edge TPU and enable this',
+        'description': 'Connect the Coral Edge TPU and ensure this is enabled in order to use Edge TPU',
```
Comment (AmandeepArora) [inline python/fledge/plugins/south/person_detection/person_detection.py:81]: May be change to "Connect Coral Edge TPU and select this option to use TPU for inference".

File: python/fledge/plugins/south/person_detection/person_detection.py [hunk 5]
```diff
@@ -59,28 +59,28 @@ def check_background():
...
         'type': 'boolean',
-        'default': 'false',
+        'default': 'true',
```
Comment (AmandeepArora) [inline python/fledge/plugins/south/person_detection/person_detection.py:83]: I think default should be false? What happens if we set this true here and TPU is not attached?

File: python/fledge/plugins/south/person_detection/person_detection.py [hunk 6]
```diff
@@ -94,22 +94,22 @@ def check_background():
         'displayName': 'Minimum Confidence Threshold'
     },
     'camera_id': {
-        'description': 'The number associated  with your video device. See /dev in your '
+        'description': 'The number associated with your video device. See /dev in your '
```
Comment (AmandeepArora) [inline python/fledge/plugins/south/person_detection/person_detection.py:97]: May be re-written as "Enter 0 to use /dev/video0, and so on."

File: python/fledge/plugins/south/person_detection/person_detection.py (hunk 5 above)
Comment (praveen-garg) [reply python/fledge/plugins/south/person_detection/person_detection.py:83]: This is as per FOGL-4142, *look the google lab manual needs. We can discuss over this and may update in next release / develop.

**Comment by praveen-garg** [comment]
Thanks @AmandeepArora! @deepanshu-yadav can you please incorporate this feedback in a new branch over develop or #2

**Comment by deepanshu-yadav** [comment]
> Thanks @AmandeepArora! @deepanshu-yadav can you please incorporate this feedback in a new branch over develop or #2

Okay @praveen-garg  @AmandeepArora  incorporating it in [this pull request](https://github.com/fledge-iot/fledge-south-human-detector/pull/6).

**Pull request merged**